### PR TITLE
Add missed table

### PR DESCRIPTION
**Pull request by bradanton**

### CODE REVIEW

**Comment by bradanton** [comment]
@clairblacketer In current implementation we moves microbiology records to the observation table, but concept_id = 4155370 has domain_id='Measurement', does this need to be changed?

**Comment by clairblacketer** [comment]
@bradanton that is a good point, let's use the concept_id 4252364 which is an observation for Clinical Microbiology

File: docs/OPTUM_PANTHER/Optum_Panther_Microbiology_STEM.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+| domain_id | Observation | | |
+| person_id | ptid | | |
+| visit_occurrence_id | encid | Lookup the VISIT_OCCURRENCE_ID based on the encid |If encid is blank then use coalesce(collect_date, result_date) to determine which VISIT_OCCURRENCE_ID the record should be associated to|
+| visit_detail_id| encid | Lookup the VISIT_DETAIL_ID based on the encid|If encid is blank then leave VISIT_DETAIL_ID blank|
+| provider_id |  encid | Lookup the PROVIDER_ID from the VISIT_DETAIL table using the encid|If encid is blank then leave PROVIDER_ID blank|
+| start_date | coalesce(collect_date, result_date)  | | |
+| end_date | coalesce(collect_date, result_date) | | | 
+| start_datetime | coalesce(collect_date, result_date) collect_time | Combine the coalesce(collect_date, result_date) and collect_time to create a datetime| |
+| end_datetime | coalesce(collect_date, result_date) collect_time | Combine the coalesce(collect_date, result_date) and collect_time to create a datetime| |
+| concept_id | 4155370 | Microbiology | |
```
Comment (clairblacketer) [inline docs/OPTUM_PANTHER/Optum_Panther_Microbiology_STEM.md:26]: Let's change this to 4252364 - Clinical Microbiology

**Review by clairblacketer — APPROVED** [review]
Let's change that concept id otherwise it's good to go

File: docs/OPTUM_PANTHER/Optum_Panther_Microbiology_STEM.md (hunk 1 above)
Comment (bradanton) [reply docs/OPTUM_PANTHER/Optum_Panther_Microbiology_STEM.md]: concept_id was updated

**Pull request merged**